pkg/handlers/scripts/install.sh

```diff
@@ -105,7 +142,24 @@ function install {
 	fi
 	#move into PATH or cwd
 	chmod +x $TMP_BIN || fail "chmod +x failed"
-	mv $TMP_BIN $OUT_DIR/kubectl-$PROG 2>/dev/null || sudo mv $TMP_BIN $OUT_DIR/kubectl-$PROG || fail "mv failed" #FINAL STEP!
+	# read installation path
+	read_installation_path
+	echo "Installing to $OUT_DIR"
```

Comment:
Same as above. This is noisy now. If they just entered the path, I don't think we need to print it out again.